tt_metal/hw/firmware/src/brisc.cc

```diff
@@ -246,7 +276,11 @@ inline void set_ncrisc_kernel_resume_deassert_address() {
     DEBUG_STATUS("INW");
     while (mailboxes->ncrisc_halt.resume_addr == 0);
     DEBUG_STATUS("IND");
+#ifdef ARCH_BLACKHOLE
+    WRITE_REG(RISCV_DEBUG_REG_NCRISC_RESET_PC, mailboxes->ncrisc_halt.resume_addr);
```

Comment:
this is fine to keep for now if it enables compilation but this will be modified in my changes to remove iram flow for ncrisc fyi @pgkeller